js/header_dialog.js

```diff
@@ -28,6 +28,7 @@ function HeaderDialog(dialog, onSave) {
             {name:'airmode_activate_throttle'	, type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'999.9.9'},
             {name:'rollPitchItermIgnoreRate'	, type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'3.0.1'},
             {name:'yawItermIgnoreRate'			, type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'3.0.1'},
+            {name:'vbat_pid_compensation'       , type:FIRMWARE_TYPE_BETAFLIGHT,  min:'2.8.0', max:'4.2.0'},
```

Comment:
imho it should be 4.2.9 - to include maintenance releases as well

Reply:
Sure @thelsotopes, thanks!

Reply:
Good point. Or even 4.2.999 to follow the pattern that is used.

Reply:
bad idea ... the pattern is 999.9.9 not 999.9.999

Reply:
yes, 999 is used only for the first number, done!

Reply:
This is broken now - it will fail for 4.2.10 and following. Please learn how to use SemVer.

Reply:
Yes right @mikeller, then im going to use 999 like you suggested